### PR TITLE
Feature 1694 orcid authentication

### PR DESCRIPTION
**Pull request by artntek**
first PR to merge changes back to dou-develop

### CODE REVIEW

File: src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java [hunk 1]
```diff
@@ -132,94 +138,69 @@ public void configureAuth(HttpServletRequest request,
...
-				SortedMap<Integer, MetaDataProperty> filePropertyMap = authMetaData
-						.getPropertiesInGroup(2);
-				Set<Integer> filePropertyIndexes = filePropertyMap.keySet();
-				for (Integer filePropertyIndex : filePropertyIndexes) {
-					String filePropertyKey = filePropertyMap.get(
-							filePropertyIndex).getKey();
-					PropertyService.checkAndSetProperty(request,
-							filePropertyKey);
+					String globalPropertyKey = globalPropertyMap.get(globalPropertyIndex).getKey();
+					PropertyService.checkAndSetProperty(request, globalPropertyKey);
```
Comment (artntek) [inline src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java:172]: checkAndSetProperty deprecated

File: src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java [hunk 2]
```diff
@@ -132,94 +138,69 @@ public void configureAuth(HttpServletRequest request,
...
 					PropertyService.persistAuthBackupProperties();
 				}
-			
+
 			} catch (GeneralPropertyException gpe) {
-				String errorMessage = "AuthAdmin.configureAuth - Problem getting or setting property while "
-					+ "processing LDAP properties page: " + gpe.getMessage();
+				String errorMessage =
+					"AuthAdmin.configureAuth - Problem getting or setting property while "
+						+ "processing LDAP properties page: " + gpe.getMessage();
```
Comment (artntek) [inline src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java:199]: `LDAP`

File: src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java [hunk 3]
```diff
@@ -229,69 +210,67 @@ public void configureAuth(HttpServletRequest request,
...
 		Vector<String> errorVector = new Vector<String>();
-
 		String adminUsers = request.getParameter("auth.administrators");
 		Vector<String> adminUserList = StringUtil.toVector(adminUsers, ':');
 
-		try {
-			AuthSession authSession = new AuthSession();
+		// Ensure that user has supplied an ID
+		if (adminUserList.size() == 0) {
+			errorVector.add("AuthAdmin.validateOptions - Invalid ORCID: Cannot be empty.");
```
Comment (artntek) [inline src/edu/ucsb/nceas/metacat/admin/AuthAdmin.java:259]: Invalid ORCID: Cannot be empty. ->  ORCID cannot be empty.

**Comment by doulikecookiedough** [comment]
Reviewed together with Matthew. Smaller PRs to improve code reviews will occur at `feature-1694-dou-develop`

**Pull request merged**